vllm/worker/model_runner.py

```diff
@@ -309,7 +314,8 @@ def _prepare_decode(
         use_captured_graph = (
             not self.model_config.enforce_eager
             and batch_size <= _BATCH_SIZES_TO_CAPTURE[-1]
-            and max_context_len <= self.max_context_len_to_capture)
+            and max_context_len <= self.max_context_len_to_capture
+            and not self.is_neuron)
```

Comment:
In config, can we just set `enforce_eager` for neuron? This can help reduce the number of `is_neuron` in the code.

Reply:
changed to set `enforce_eager` for neuron support.